eng/testing/tests.wasm.targets

```diff
@@ -49,6 +50,11 @@
     <_ExtraTrimmerArgs Condition="'$(WasmEnableSIMD)' == 'true' and '$(RunAOTCompilation)' == 'true'">$(_ExtraTrimmerArgs) --substitutions &quot;$(MonoProjectRoot)\wasm\build\ILLink.Substitutions.WasmIntrinsics.xml&quot;</_ExtraTrimmerArgs>
     <_ExtraTrimmerArgs Condition="'$(WasmEnableSIMD)' != 'true' or '$(RunAOTCompilation)' != 'true'">$(_ExtraTrimmerArgs) --substitutions &quot;$(MonoProjectRoot)\wasm\build\ILLink.Substitutions.NoWasmIntrinsics.xml&quot;</_ExtraTrimmerArgs>
   </PropertyGroup>
+  <ItemGroup Condition="'$(BuildAOTTestsOn)' == 'helix'">
+    <TrimmerRootDescriptor
+      Condition="'$(WasmEnableLegacyJsInterop)' == 'true'"
+      Include="$(MonoProjectRoot)\wasm\build\ILLink.Descriptors.LegacyJsInterop.xml" />
+  </ItemGroup>
```

Comment:
IIUC, this shouldn't be needed because the xml will be included in the runtime pack, and will thus be in the workload, and get used automatically by WasmApp.targets . But you do need to add `WasmEnableLegacyJsInterop` to `_WasmPropertyNames` in `_BundleAOTTestWasmAppForHelix` target.

Reply:
When this was not here it failed the test because 'LegacyExports' were missing. Note radek's SIMD has simmilar trimming stuff few lines above. But his stuff could be trimmed later, my stuff just disapears and there is no way back.

Reply:
It is trimmed already on CI not on helix. And it should not be. 

`_WasmPropertyNames` is good advice.

Reply:
Hm I wrote that block above, but now I'm feeling like the condition isn't quite correct. I will try it out.

As for your block, it's fine for now. I will try to improve the situation in the future so these hacks aren't needed.